diff --git a/dist/index.js b/dist/index.js
index c81c9e1a36a18c5c3960352f10ca399f35d14275..86f503167becc6030e6b03e722b0329350a676b7 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1777,6 +1777,11 @@ const remarkPluginNormalizeLink = ({ root, cleanUrls, routeService })=>(tree, fi
             const relativePath = __WEBPACK_EXTERNAL_MODULE_node_path_c5b9b54f__["default"].relative(root, file.path);
             node.url = normalizeLink(nodeUrl, routeService, relativePath, cleanUrls);
         });
+        (0, __WEBPACK_EXTERNAL_MODULE_unist_util_visit_555e002a__.visit)(tree, 'definition', (node)=>{
+            const { url: nodeUrl } = node;
+            const relativePath = __WEBPACK_EXTERNAL_MODULE_node_path_c5b9b54f__["default"].relative(root, file.path);
+            node.url = normalizeLink(nodeUrl, routeService, relativePath, cleanUrls);
+        });
         const getMdxSrcAttribute = (tempVar)=>({
                 type: 'mdxJsxAttribute',
                 name: 'src',
diff --git a/dist/loader.js b/dist/loader.js
index 69d05932cb786d6053d803decaa16d65d7294cb7..67fcbeb9460f566a5ac8080e9365e207dc75fde1 100644
--- a/dist/loader.js
+++ b/dist/loader.js
@@ -216,6 +216,11 @@ const remarkPluginNormalizeLink = ({ root, cleanUrls, routeService })=>(tree, fi
             const relativePath = __WEBPACK_EXTERNAL_MODULE_node_path_c5b9b54f__["default"].relative(root, file.path);
             node.url = normalizeLink(nodeUrl, routeService, relativePath, cleanUrls);
         });
+        (0, __WEBPACK_EXTERNAL_MODULE_unist_util_visit_555e002a__.visit)(tree, 'definition', (node)=>{
+            const { url: nodeUrl } = node;
+            const relativePath = __WEBPACK_EXTERNAL_MODULE_node_path_c5b9b54f__["default"].relative(root, file.path);
+            node.url = normalizeLink(nodeUrl, routeService, relativePath, cleanUrls);
+        });
         const getMdxSrcAttribute = (tempVar)=>({
                 type: 'mdxJsxAttribute',
                 name: 'src',
